app/src/main/java/org/oppia/android/app/walkthrough/WalkthroughVerticalScrollListener.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.android.app.walkthrough
+
+/** Interface to show or hide the Progress Bar and Header Text. */
+interface WalkthroughVerticalScrollListener {
```

Comment:
Per the discussion during the team meeting this week, do we still need this if we go with a collapsible header?

Reply:
Probably no, actually, I assigned you the PR before the meeting.